dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -783,6 +784,23 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 										/>
 									</div>
 								)}
+
+								{CAPIArticle.config.switches.automaticFilters &&
```

Comment:
Do you think its worth extracting these to a variable and adding a note to explain what's going on? Might make it easier to unpick when the time comes

Reply:
Good idea, I've updated with a comment for clarity